index.d.ts

```diff
@@ -4,4 +4,14 @@ export interface JwtDecodeOptions {
   header?: boolean;
 }
 
-export default function jwtDecode(token: string, options?: JwtDecodeOptions): unknown;
+export interface JwtClaims {
+  iss?: string;
+  sub?: string;
+  aud?: string[] | string;
+  exp?: number;
+  nbf?: number;
+  iat?: number;
+  jti?: string;
+}
+
+export default function jwtDecode(token: string, options?: JwtDecodeOptions): JwtClaims;
```

Comment:
I'd say this is a really easy way to make it generic without changing too much
```suggestion
export default function jwtDecode<ReturnType = unknown>(token: string, options?: JwtDecodeOptions): ReturnType;
```